plotting/plot_factory/dashapp.py

```diff
@@ -9,10 +9,11 @@
 """
 
 # Core Dependencies
-import dash
 import dash_core_components as dcc
 import dash_html_components as html
 
+from django_plotly_dash import DjangoDash
+
 # pylint: disable=too-few-public-methods
 class DashApp:
     """Creates a DashApp for direct insertion into a web page"""
```

Comment:
_nit_
Might be good to go through and make all docstring consistent, e.g.
- spaces after and before the `"""`s
- a space after colons `:`